README.md

```diff
@@ -118,6 +118,23 @@ void main() async {
 }
 ```
 
+## Repro Steps
+Repro Steps list all of the actions an app user took before reporting a bug or crash, grouped by the screens they visited in your app.
+ 
+ To enable this feature, you need to add `InstabugNavigatorObserver` to the `navigatorObservers` :
+ ```
+  runApp(MaterialApp(
+    title: ...,
+    navigatorObservers: [InstabugNavigatorObserver()],
```

Comment:
I understand the other keys are not relevant, no? I suggest replacing the other attributes with a comment instead; the snippet would be compilable as well.